### ISSUE TITLE
Error with Storybook 6.5.0-alpha.64

### ISSUE THREAD

**Issue by maddesigns** [chanzuckerberg/axe-storybook-testing#51]
Again, early adopter of the new Storybook 6.5.0-alpha.64 version.

I’m getting the following error:
```
Error! The suite failed to run
An error was encountered before we started testing stories. Likely this means the browser failed to open.

Error: page.evaluate: Error: Cannot call extract() unless you call cacheAllCSFFiles() first.
    at StoryStore.extract (file:///Users/svenwolfermann/Sites/aperto/drv-design-system/storybook-static/4020.885ff43e.iframe.bundle.js:2:187701)
    at StoryStore.raw (file:///Users/svenwolfermann/Sites/aperto/drv-design-system/storybook-static/4020.885ff43e.iframe.bundle.js:2:188867)
    at checkStories (eval at evaluate (:3:2389), <anonymous>:6:65)
    at eval (eval at evaluate (:3:2389), <anonymous>:26:5)
    at new Promise (<anonymous>)
    at fetchStoriesFromWindow (eval at evaluate (:3:2389), <anonymous>:2:10)
    at t.default.evaluate (<anonymous>:3:2412)
    at t.default.<anonymous> (<anonymous>:1:44)
```

Can wait for the stable release, just want to let you know.

**Comment by ahuth** [comment]
@maddesigns, I actually can't replicate this. Still seems to work with 6.5.0-alpha.64 in my testing.

Any chance you're setting `storyStoreV7`?

If so, I can fix that. But unfortunately the fix is affected by https://github.com/storybookjs/storybook/issues/16967. Once that's resolved I'll push up a fix.

**Comment by ahuth** [comment]
Just realized I can pass some command line flags through Playwright working around the storybook issue. Should have a fix for this out soon.

**Comment by maddesigns** [comment]
Thanks for investigating @ahuth! Indeed I’m using the `storyStoreV7`. It’s a great feature, but I can live without it for now. 
Not using the `storyStoreV7` fixing the issue. :)

**Comment by ahuth** [comment]
@maddesigns, looks like the [official storyshots addon has a (possibly) related issue](https://github.com/storybookjs/storybook/issues/16939).

The [recommended workaround for storyshots](https://github.com/storybookjs/storybook/blob/3064e536ece1ec602823cff76c61c7473cf621d1/MIGRATION.md#storyshots-compatibility-in-the-v7-store) won't quite work here, but it made me realize you could try something like

```js
// In .storybook/main.js
module.exports = {
  features: {
    storyStoreV7: process.env.NO_STORY_STORE_V7 !== 'true',
  },
};
```

and then set the env variable in npm script running axe-storybook-testing

```
// In package.json
"scripts": {
  "storybook:axe": "NO_STORY_STORE_V7=true build-storybook && axe-storybook"
},
```

This would let you use storyStoreV7 when running Storybook, but disable it when running this project. At least until I can figure out a long term solution.

**Comment by ahuth** [comment]
@maddesigns this should be fixed in v5.0.1. Let me know if that solves it for you or not.
